vmware-event-router/internal/config/v1alpha1/metrics.go

```diff
@@ -0,0 +1,20 @@
+package v1alpha1
```

Comment:
What metrics server do we use? Do we have new metrics server deployed with VEBA, or the Prometheus one provided by default by the OF bundle?

Reply:
The README has the details on the available metrics providers. For now the `default` is an "internal" http metrics provider exposing some JSON data under the `/stats` endpoint.